corehq/apps/sms/static/sms/js/chat_contacts.js

```diff
@@ -23,21 +23,23 @@ var initialPageData = hqImport('hqwebapp/js/initial_page_data');
             "aoColumnDefs": [
                 {
                     "aTargets": [0],
-                    "render": function(data, type, row, meta) {
-                        return '<a target="_blank" href="'+row[4]+'">'+row[0]+'</a>'+
-                               '<span class="btn btn-primary pull-right" '+
-                               'onClick="window.open(\''+row[5]+'\', \'_blank\', \'location=no,menubar=no,scrollbars=no,status=no,toolbar=no,height=400,width=400\');">' +
+                    "render": function(data, type, row) {
+                        return '<a target="_blank" href="' + row[4] + '">' + row[0] + '</a>' +
```

Comment:
We sometimes use underscore templating to clean up this kind of HTML concatenation nastiness, like [this](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/static/app_manager/js/section_changer.js#L40-L43).

Zero obligation to do this in this PR or elsewhere, just fyi in case you haven't seen it yet.

Reply:
That's much nicer. I was missing the ES6 templating notation, but this is good too